src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_NullablePublicOnly.cs

```diff
@@ -180,20 +200,182 @@ public class B : A<object?>
             CompileAndVerify(comp, symbolValidator: AssertNullablePublicOnlyAttribute);
         }
 
-        private static void AssertNoNullablePublicOnlyAttribute(ModuleSymbol module)
+        [Fact]
+        public void NullableDisabled_OtherNullableAttributeDefinitions()
         {
-            AssertAttributes(module.GetAttributes());
+            var source =
+@"public class Program
+{
+}";
+            var options = WithNonNullTypesFalse().WithMetadataImportOptions(MetadataImportOptions.All);
+            var parseOptions = TestOptions.Regular8;
+            CSharpTestSource sources = new[] { NullableAttributeDefinition, NullableContextAttributeDefinition, source };
+
+            var comp = CreateCompilation(sources, options: options, parseOptions: parseOptions);
+            CompileAndVerify(comp, symbolValidator: AssertNoNullablePublicOnlyAttribute);
+
+            comp = CreateCompilation(sources, options: options, parseOptions: parseOptions.WithFeature("nullablePublicOnly"));
```

Comment:
>parseOptions.WithFeature("nullablePublicOnly") [](start = 78, length = 46)

Consider making a TestOptions option for these, so we don't have to keep typing the flag out.

Reply:
Will add that in a subsequent PR.

---
In reply to: [301323414](https://github.com/dotnet/roslyn/pull/37019#discussion_r301323414) [](ancestors = 301323414)